src/mongoose_async_pools.erl

```diff
@@ -77,6 +82,8 @@ start_link(HostType, PoolId, Opts) ->
 -spec init(term()) -> {ok, {supervisor:sup_flags(), [supervisor:child_spec()]}}.
 init([HostType, PoolId, Opts]) ->
     PoolName = gen_pool_name(HostType, PoolId),
+    mongoose_metrics:ensure_metric(HostType, [?MODULE, PoolId, timed_flushes], counter),
```

Comment:
can we have a test that it gets incremented?

Reply:
Metric test implemented